backend/src/middleware/checkObjectId.js

```diff
@@ -0,0 +1,10 @@
+import mongoose from 'mongoose';
+
+const checkObjectId = (idToCheck) => (res, next) => {
```

Comment:
We can skip this check to shed a few milliseconds. The ObjectId will be valid since it's coming from the auth token.

Reply:
I removed the ObjectId validator